docs/sources/writing-guide/image-guidelines/index.md

```diff
@@ -255,13 +255,28 @@ It is important that you generate a local build of your docs so that you can ver
 
    Store image files in the following website repo directory: `static/static/img/docs`. If a relevant sub-directory doesn't exist, you can create it.
 
-1. After the Website team merges the image PR into the website repo, add the following figure shortcode to your docs: 
+1. After the Website team merges the image PR into the website repo, add the following figure shortcode to your docs:
 
    `{{</* figure src="[path to the image in the website repo]" */>}}`
-   
+
    **Example:**
-   - Path to image stored in the website repo: `static/static/img/docs/grafana-cloud/k8s-node-capacity.png` 
+   - Path to image stored in the website repo: `static/static/img/docs/grafana-cloud/k8s-node-capacity.png`
    - Corresponding shortcode: `{{</* figure src="/static/img/docs/grafana-cloud/k8s-node-capacity.png" */>}}`.
 
 1. Run `make docs` on your branch and verify that the image appears.
 1. (Optional) Make adjustments to the image in the website repo and test again.
+
+
+## Videos
+
+The Creative Services team periodically creates videos for blog posts and other collateral. Most of these videos are hosted on Vimeo.
+
+You can embed a Vimeo-hosted video by using the `vimeo` short code and the video number:
```

Comment:
Could you show the (unevaluated) shortcode for this example?  I guess you could just put
` {{< vimeo 773194063 >}}` 
on a line by itself like in the figure example in the previous section?

Reply:
Apparently, you can put the video link on it's own line or at the end of a sentence. It floats to where it fits.